torch/fx/traceback.py

```diff
@@ -1,80 +1,63 @@
 import traceback
 from contextlib import contextmanager
-from typing import Optional, List, Any, Dict
+from typing import List, Any, Dict
 from ._compatibility import compatibility
 
-__all__ = ['override_stack_trace', 'set_stack_trace', 'append_stack_trace', 'format_stack',
-           'is_stack_trace_overridden', 'get_current_meta', 'set_current_meta']
+__all__ = ['preserve_node_meta', 'has_preserved_node_meta',
+           'set_stack_trace', 'format_stack',
+           'set_current_meta', 'get_current_meta']
 
-
-current_stack: List[str] = []
 current_meta: Dict[str, Any] = {}
-is_overridden = False
+is_enabled = False
 
 
 @compatibility(is_backward_compatible=False)
 @contextmanager
-def override_stack_trace():
-    global is_overridden
+def preserve_node_meta():
```

Comment:
nit: maybe rename this to `should_preserve_node_meta` to make it clearer that this is a boolean?